etna/analysis/change_points_trend/find_change_points.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict
+from typing import List
+
+import numpy as np
+import pandas as pd
+from ruptures.base import BaseEstimator
+from ruptures.costs import CostLinear
+
+from etna.datasets import TSDataset
+
+
+def _prepare_signal(model: BaseEstimator, series: pd.Series) -> np.ndarray:
+    """Prepare series for change point model."""
+    signal = series.to_numpy()
+    if isinstance(model.cost, CostLinear):
+        signal = signal.reshape((-1, 1))
+    return signal
+
+
+def _find_change_points_segment(
+    series: pd.Series, change_point_model: BaseEstimator, **model_predict_params
+) -> List[pd.Timestamp]:
+    """Find change trend points within one segment."""
+    signal = _prepare_signal(series=series, model=change_point_model)
```

Comment:
Make the order of parameters consistent with the order of parameters in signature(or vice versa)